www/_template/concepts/build-pipeline.md

```diff
@@ -3,38 +3,39 @@ layout: layouts/content.njk
 title: The Build Pipeline
 description: Snowpack Build creates a production-ready website with or without a bundler
 ---
+THe Snowpack build pipeline runs on the command `snowpack build`. Snowpack then creates a production-ready website, with or without a bundler.
 
-![build output example](/img/snowpack-build-example.png)
+This article will cover
+- What Snowpack does by default
+- When and how to use a bundler
 
-`snowpack build` - When you're ready to deploy your application, run the build command to generate a static production build of your site. Building is tightly integrated with your dev setup so that you are guaranteed to get a near-exact copy of the same code that you saw during development.
+Snowpack’s philosophy is **you should be able to use a bundler because you want to, and not because you need to.** Snowpack treats bundling as an optional production optimization, which means you're free to skip the extra complexity of bundling until you need it.
```

Comment:
can strike "be able to"

Reply:
also, it feels strange that the intro is all about not using a bundler, and then the first section is "using a bundler". Can you move this paragraph and the section below it into something like "Snowpack's default build" or "the build pipeline"? 

Also, the next paragraph could use more detail. What unbundled approach? Why is that all you need? The old paragraph that started with "By default"  had some important info that got removed that explained this in more detail.